app/views/shared/_top_breadcrumbs.html.erb

```diff
@@ -20,7 +20,14 @@
 
         <% if @heading && !@heading.declarable?%>
           <li class="govuk-breadcrumbs__list-item">
-            <%= breadcrumb_link_or_text(@heading, @commodity, "Heading #{@heading.short_code}") %>
+            <% child = @commodity || @subheading %>
```

Comment:
to remove the assignment, what about just using the value (I think it is not worth adding a helper for that)

```ruby
breadcrumb_link_or_text(@heading,
  @commodity || @subheading,
  "Heading #{@heading.short_code}
)
```

Reply:
What's wrong with the assignment?